### PR TITLE
Added changelog

### LINKED ISSUE #328: Adding Changelog with version release and release notes
We do not have the changelogs for this project yet.

## Description
It would help developers to precisely see what notable changes have been made between each release of the project.

### PR DESCRIPTION
**Pull request by Anushka-shukla**
Added Changelog

## Description
We did not have the changelogs for this project with version release and release notes and all the additions, fixes, and changes that were made with each release.

## Motivation and Context
It would help developers to precisely see what notable changes have been made between each release of the project.

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +159,22 @@ look like a commit ID. Note that this might break previous permalinks.
...
+[NOTICE.md](NOTICE.md) for required attribution notices.
+
+## Contribute
+
+Feel free to contribute! You may [raise an issue](https://github.com/common-workflow-language/cwlviewer/issues),
+provide a [pull request](https://github.com/common-workflow-language/cwlviewer/pulls)
+or join the [gitter chat for cwlviewer](https://gitter.im/common-workflow-language/cwlviewer)!
+
+## Changelog
+See [Changes](https://github.com/common-workflow-language/cwlviewer/blob/main/CHANGELOG.md)
```
Comment (tetron) [inline README.md:176]: Can this be made into a relative link, like?  `[Changes](CHANGELOG.md)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- Better support for packed workflows
+    *  Use %26 (#) in the URL to view individual workflows as per cwltool
+    * Links to subworkflows within a packed file now function correctly
+- Improved error reporting
+- Fix for array and optional types not being parsed correctly
+- Fix for twitter meta tags not containing the correct image URL
+- Fix for a concurrency issue
+- Brings API documentation up to date with the current state of the JSON API
+
+## v1.2.0 [22 Aug 2017]
```
Comment (tetron) [inline CHANGELOG.md:37]: This date format is inconsistent with the others

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- [Permalinks](https://github.com/common-workflow-language/cwlviewer/pull/175) with content negotiation
+- Show workflow [license](https://github.com/common-workflow-language/cwlviewer/pull/198)
+- Added [Privacy Policy](https://github.com/common-workflow-language/cwlviewer/pull/194)
+### Fixed
+- Support packed workflows in [permalinks](https://github.com/common-workflow-language/cwlviewer/pull/177)
+- Use [Python 3 and nodejs](https://github.com/common-workflow-language/cwlviewer/pull/197)
+- Support [git tags](https://github.com/common-workflow-language/cwlviewer/pull/192) 
+- UI: [Shrink description text](https://github.com/common-workflow-language/cwlviewer/pull/178)
+
+## v1.2.2 [ 24-08-2017]
```
Comment (tetron) [inline CHANGELOG.md:18]: Fix extra space after `[`

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+### Fixed
+- Fix for invalid branch names being accepted in some circumstances
+- Fix regression in supporting slashes in branch names
+- [References](https://doi.org/10.5281/zenodo.848163)
+
+## v1.2.1 [ 22-08-2017]
```
Comment (tetron) [inline CHANGELOG.md:24]: Fix extra space after `[`

**Comment by Anushka-shukla** [comment]
@tetron I've made all the changes.

File: README.md [hunk 5]
```diff
@@ -210,6 +197,22 @@ look like a commit ID. Note that this might break previous permalinks.
...
+
+Distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+See the file [LICENSE.md](LICENSE.md) for details, and
+[NOTICE.md](NOTICE.md) for required attribution notices.
+
+## Contribute
+
+Feel free to contribute! You may [raise an issue](https://github.com/common-workflow-language/cwlviewer/issues),
+provide a [pull request](https://github.com/common-workflow-language/cwlviewer/pulls)
+or join the [gitter chat for cwlviewer](https://gitter.im/common-workflow-language/cwlviewer)!
```
Comment (kinow) [inline README.md:211]: Include a link to `CONTRIBUTING.md` here?

**Review by kinow — APPROVED** [review]
:+1: not sure if the Changelog needs updating since the PR was created, but great work @Anushka-shukla for writing it. One comment on including a link to `CONTRIBUTING.md` under the contributions section, but looks good to me other than that. Thanks!

**Review by mr-c — APPROVED** [review]

**Pull request merged**